src/StructureElementConnectivityRestraint.cpp

```diff
@@ -22,20 +22,64 @@ StructureElementConnectivityRestraint::StructureElementConnectivityRestraint(Mod
 
 /* Apply the pair score to each particle pair listed in the container.
  */
-
+static float ll_means_[90] = {3.81, 3.036, 2.836, 2.511, 2.275, 2.178, 2.026, 1.876, 1.835, 1.669, 1.658, 1.666, 1.625, 1.53,
```

Comment:
Generally not a great idea to have random tables of magic numbers in code without some sort of comment about where they come from. Although it looks like you've inherited this from Daniel.

It's not clear to me on a cursory reading of the code what these tables are for, but if they need to be in way configurable, generally a better idea to store them in a text file somewhere and read them in at runtime. Every IMP module has a data directory for this purpose.